var/spack/repos/builtin/packages/catalyst/package.py

```diff
@@ -0,0 +1,154 @@
+##############################################################################
+# Copyright (c) 2017 Simone Bna, CINECA.
```

Comment:
As an aside, I don't know if a PR where you hold the copyright can be accepted here. Summoning @scheibelp or @tgamblin to have more information on this :smile:

Reply:
Hi @alalazo, thanks for your suggestions. I will contact tgamblin about the copyright issue.

Reply:
Hi @tgamblin, @scheibelp,
is it possible to set the copyright as above in a package.py file of a PR?

Reply:
@alalazo CI fails with the following message:
The command "pip install --upgrade codecov" failed and exited with 1 during .

Reply:
@sbna I restarted the build that errored, and now everything seems fine. 

@tgamblin There's something fishy going on with coverage. Here we have a +0.35% on `core`, and as far as I can see @sbna just committed a package. Pinging @adamjstewart to keep him in the loop.